src/assisted_test_infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -174,8 +174,11 @@ def _fill_tfvars(self, running=True):
         tfvars["bootstrap_in_place"] = self._config.bootstrap_in_place
 
         vips = self.get_ingress_and_api_vips()
-        tfvars["api_vip"] = vips["api_vip"]
-        tfvars["ingress_vip"] = vips["ingress_vip"]
+        tfvars["api_vip"] = self._config.api_vip or vips["api_vip"]
```

Comment:
In case of day2 cluster, we want to set the vips of the existing day1 cluster, which are set in `self._config.api_vip` and `self._config.ingress_vip`. 
If `self._config.ingress_vip` and `self._config.api_vip` are not set, then we let `get_ingress_and_api_vips()` select vips for us.

Reply:
They are set [here](https://github.com/openshift/assisted-test-infra/pull/2009/files#diff-132040bf88e1eaf0881057dc6ba2099051c758b0c01cd7f2fe09d40eaeb20577R147-R148) while building the controller configuration

Reply:
I could move it logic, in `get_ingress_and_api_vips()` though